web/src/views/source/panels.ts

```diff
@@ -30,71 +31,81 @@ class EditorPanelElement extends TWLitElement {
+      'flex justify-between',
+      'h-6',
+      'px-4 py-0.5',
+      'bg-gray-wild-sand',
+    ])
+
     return html`
-      <div class="p-2 flex justify-between">
+      <div class=${styles}>
         ${this.renderControls()} ${this.renderBreadcrumbs()}
       </div>
     `
   }
 
   private renderControls() {
+    const styles = apply([
+      'flex flex-row items-center justify-start',
+      'text-sm',
+    ])
+
     return html`
-      <div
-        class="flex flex-row items-center justify-start bg-brand-white text-sm"
-      >
-        <div class="mr-2">${this.renderFormatSelect()}</div>
-        <div>${this.renderLineWrapCheckbox()}</div>
+      <div class=${styles}>
+        ${this.renderLineWrapButton()} ${this.renderFormatSelect()}
       </div>
     `
   }
 
   private renderFormatSelect() {
+    const changeEvent = (e: Event) =>
+      (this.sourceView.format = (e.target as HTMLSelectElement).value)
+
+    const styles = apply(['w-28 h-full', 'mx-3', 'bg-white', 'rounded-sm'])
+
     return html`
-      <label>
-        Format
-        <select
-          @change=${(e: Event) =>
-            (this.sourceView.format = (e.target as HTMLSelectElement).value)}
-        >
-          ${Object.entries(FORMATS).map(
-            ([format, name]) =>
-              html`<option
-                value=${format}
-                ?selected=${this.sourceView.format === format}
-              >
-                ${name}
-              </option>`
-          )}
-        </select>
-      </label>
+      <select title="Document format" class=${styles} @change=${changeEvent}>
+        ${Object.entries(FORMATS).map(
+          ([format, name]) =>
+            html`<option
+              value=${format}
+              ?selected=${this.sourceView.format === format}
+            >
+              ${name}
+            </option>`
+        )}
+      </select>
     `
   }
 
-  private renderLineWrapCheckbox() {
+  private renderLineWrapButton() {
+    const clickEvent = () => {
+      this.sourceView.lineWrap = !this.sourceView.lineWrap
+    }
+    const title = `Toggle line wrapping: ${this.sourceView.lineWrap ? 'ON' : 'OFF'}`
+    const styles = apply([
+      'h-4 w-4',
+      this.sourceView.lineWrap ? 'bg-gray-200' : '',
+      'hover:bg-green-000',
+    ])
+
     return html`
-      <label>
-        ${'Enable line wrapping'}
-        <input
-          type="checkbox"
-          class="ml-1"
-          ?checked="${this.sourceView.lineWrap}"
-          @change="${(e: Event) =>
-            (this.sourceView.lineWrap = (
-              e.target as HTMLInputElement
-            ).checked)}"
-        />
-      </label>
+      <button class=${styles} title=${title} @click=${clickEvent}>
+        <img src=${icon} width="100%" height="100%" />
+      </button>
     `
   }
 
   private renderBreadcrumbs() {
     return html`
-      <div>
+      <div class="text-xs leading-none flex items-center">
         ${this.breadcrumbs.reverse().map((entry, i, arr) => {
```

Comment:
```suggestion
        ${this.breadcrumbs.reverse().slice(1).map((entry, i, arr) => {
```

Don't include the first `Article` in the breadcrumb its redundant.